script/Arcueid/Day 1/01_00_ARC01_8.txt

```diff
@@ -67,7 +67,7 @@ The white walls are reminiscent of castle ramparts.
 [sha:c12b2e20ac73967fe6ae8926d6a9f390dc8f12ed]{
 -- Page 449, Offset 2303.
 -- 温かみのあるオレンジ色の電灯が延々と続く中、メイド服の女の子が歩いている。
-Down this endless corridor of warm orange lights walks a girl in a maid uniform.
+Down this endless corridor of warm orange lights walks a girl in a maid's uniform.
```

Comment:
almost everywhere I look "maid uniform" is more common. And by a lot. Even "maid outfit" is more common

Reply:
alright sure